doc/nrf/includes/tfm_spm_thingy91.txt

```diff
@@ -1,8 +1,11 @@
 The sample is configured to compile and run as a non-secure application.
-Therefore, it automatically includes an additional secure firmware component, that prepares the required peripherals and secure services to be available for the application.
+Therefore, it automatically includes an additional secure firmware component that prepares the required peripherals and secure services to be available for the application.
 
-When built for the ``nrf9160dk_nrf9160_ns`` build target, the sample automatically includes :ref:`Trusted Firmware-M <ug_tfm>`.
+For nRF9160 DK, when the sample is built using the ``nrf9160dk_nrf9160_ns`` build target, it automatically includes :ref:`Trusted Firmware-M <ug_tfm>`.
 You can also configure it to use the :ref:`secure_partition_manager` instead of Trusted Firmware-M.
 
-When built for the ``thingy91_nrf9160_ns`` build target, the sample automatically includes the :ref:`secure_partition_manager`.
+For Thingy:91, when the sample is built using the ``thingy91_nrf9160_ns`` build target, it automatically includes the :ref:`secure_partition_manager`.
 You can also configure it to use :ref:`TF-M <ug_tfm>` instead of Secure Partition Manager.
+
+.. note::
+   TF-M is not completely supported for Thingy:91 v1.5.0 and lower versions.
```

Comment:
Do we have a known issue that we can link to? If not, should we provide more details here as to what is not supported?

Reply:
The part that is not supported in NCS v2.0.0 for Thingy:91 v1.5.0 and older is upgrading the firmware using MCUboot. I assume it's also valid for other boards with old MCUboot that doesn't do complete cleanup.
This PR fixes it for the older Thingy:91s: https://github.com/nrfconnect/sdk-trusted-firmware-m/pull/51

Reply:
@jtguggedal : I have update the note. Could you please check this.